django/river/transformer/fhir.py

```diff
@@ -12,6 +12,12 @@ def recursive_defaultdict():
     return defaultdict(recursive_defaultdict)
 
 
+def recursive_defaultdict_to_dict(rec_default_dict):
```

Comment:
what was the issue ?

Reply:
yes we should put a comment. It solves this bug https://bugs.python.org/issue35540. `dataclasses.asdict` method can't handle defaultdict. @Jasopaum wrote this to convert recusively the fhir_object from a nested defaultdict to a nested native dict.